documentation/reference/document-v1-api-reference.html

```diff
@@ -273,18 +273,27 @@ <h2 id="request-body">Request body</h2>
 
 
 <h2 id="http-status-codes">HTTP status codes</h2>
-Non-exhaustive list of status codes:
+<p>
+  See <a href="https://github.com/vespa-engine/vespa/issues/13465">13465</a>
+  for defined document types not assigned to a content cluster.
```

Comment:
I think this is too specific to put here. _Perhaps_ something to list under the description of code 400.

Reply:
agree, move there for now